POSTFIX_CERTS_GENERATION.md

```diff
@@ -0,0 +1,102 @@
+# Steps for generation self signed SSL key for postfix
+Let's create a self signed SSL key in cert directory.
+In order to do it let's create `certs` directory.
+First generate a private key for the server
+
+```bash
+$ openssl genrsa -des3 -out localhost.key 2048
```

Comment:
Why are you encrypting the key, if you are decrypting it again a few steps later? Drop -des3.

You can also squash the whole process. You can generate a key and a CSR at the same time:
openssl req -new -newkey rsa:2048 -nodes -keyout localhost.key -out localhost.csr
and then sign it, or do the whole thing in one step:
openssl req -x509 -days 365 -nodes -newkey rsa:2048 -keyout localhost.key -out localhost.crt

It would save people some time.